shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -2424,7 +2461,19 @@ export default {
               @update:value="$emit('input', $event)"
               @s3-backup-changed="handleS3BackupChanged"
               @config-etcd-expose-metrics-changed="handleConfigEtcdExposeMetricsChanged"
-            />
+            >
+              <template #s3-config>
```

Comment:
this isn't used, there's no `<slot name="s3-config">` or `$slot` management in the Etcd component